features/digital_summary.feature

```diff
@@ -12,3 +12,13 @@ Feature: Digital Appointment Summaries
       | requested_digital |
       | requested         |
       | did not request   |
+
+  Scenario: Email notification of summary document location
+    Given the customer requested a digital appointment summary
+    When I create their record of guidance
+    Then the customer should be notified
+
+  Scenario: No email notification when postal notification requested
+    Given the customer did not requested a digital appointment summary
+    When I create their record of guidance
+    Then the customer should not be notified
```

Comment:
Feels like this is getting into unit testing territory. Should this be more like:

"Then the appointment summary is sent by post"

and then we ensure it's landing in the CSV